spring-integration-groovy/src/main/java/org/springframework/integration/groovy/config/GroovyControlBusFactoryBean.java

```diff
@@ -16,22 +16,27 @@
 import java.util.HashMap;
 import java.util.Map;
 
-import groovy.lang.Binding;
-import groovy.lang.MissingPropertyException;
-import org.springframework.beans.factory.*;
+import org.springframework.beans.factory.BeanClassLoaderAware;
+import org.springframework.beans.factory.BeanCreationNotAllowedException;
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.NoSuchBeanDefinitionException;
 import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
 import org.springframework.context.Lifecycle;
 import org.springframework.core.annotation.AnnotationUtils;
-import org.springframework.messaging.Message;
 import org.springframework.integration.config.AbstractSimpleMessageHandlerFactoryBean;
-import org.springframework.messaging.MessageHandler;
 import org.springframework.integration.groovy.GroovyCommandMessageProcessor;
 import org.springframework.integration.handler.ServiceActivatingHandler;
 import org.springframework.integration.scripting.ScriptVariableGenerator;
+import org.springframework.integration.support.management.IntegrationManagedResource;
 import org.springframework.jmx.export.annotation.ManagedResource;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHandler;
 import org.springframework.scripting.groovy.GroovyObjectCustomizer;
 import org.springframework.util.CustomizableThreadCreator;
 
+import groovy.lang.Binding;
+import groovy.lang.MissingPropertyException;
```

Comment:
I will add the author and blank lines, but I think I have explained before that I do not like reformatting just for the sake of it - it disturbs the git blame, making it hard to see why some change was made (it hides the real blame).

I prefer to fix formatting only when the code in violation requires a change for the feature/bug.